lib/model/rwfolder.go

```diff
@@ -780,13 +778,9 @@ func (f *rwFolder) renameFile(source, target protocol.FileInfo) {
 	from := filepath.Join(f.dir, source.Name)
 	to := filepath.Join(f.dir, target.Name)
 
-	if f.versioner != nil {
```

Comment:
So I tend to change this now and  someone else can iterate on this later again. Do you agree?